src/cljs/nnts2/organization/events.cljs

```diff
@@ -0,0 +1,32 @@
+(ns nnts2.organization.events
+  (:require
+   [re-frame.core :as re-frame]
+   [day8.re-frame.http-fx]
+   [nnts2.user.events]
+   [nnts2.organization.api :as org-api]))
+
+(re-frame/reg-event-fx
+ :create-organization
+ (fn [{db :db}  [_ org-details]]
+   {:http-xhrio (org-api/create-map org-details)
+    :db (assoc-in db [:organization :event :create :state] :loading)}))
+
+(re-frame/reg-event-db
+ :organization-created
+ (fn [db [_ org-details]]
+   (-> db
+       (update-in [:organization :event :create :state] :created)
+       (update-in [:organization :data :all] (fnil conj []) org-details))))
```

Comment:
You should do a get orgs call at this point and not insert org in this list just after creation. It will nullify any sorting coming from backend. Side effects can be initial ordering (from the get-all-orgs on component load) conflicts with the newly added org.
Basically, avoiding ordering in frontend unless thats an explicit functionality